doc/hotspot-style.md

```diff
@@ -572,9 +572,14 @@ There are a few exceptions to this rule.
 
 * `#include <new>` to use placement `new`, `std::nothrow`, and `std::nothrow_t`.
 * `#include <limits>` to use `std::numeric_limits`.
-* `#include <type_traits>`.
+* `#include <type_traits>` with some considerations, listed below.
 * `#include <cstddef>` to use `std::nullptr_t` and `std::max_align_t`.
 
+Certain declarations from `<type_traits>` should be avoided.
```

Comment:
s/should be avoided/are forbidden/

Reply:
I'll just rewrite this one to say it has restrictions that apply to it, to match the line about it from above